.github/workflows/memory-badger-integration-tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Memory and Badger Integration Tests
```

Comment:
perhaps add this file in the PR that fixes the Badger tests, so that this PR could be merged green.